### PR TITLE
Vtcca 4236 Play 2.8 migration

### PR DESCRIPTION
**Pull request by reddy-yattapu**

### CODE REVIEW

File: app/connectors/PropertyRepresentationConnector.scala [hunk 1]
```diff
@@ -17,20 +17,20 @@
 package connectors
 
 import connectors.errorhandler.exceptions.ExceptionThrowingReadsInstances
+
 import javax.inject.Inject
 import models.propertyrepresentation.AgentDetails
-import play.api.Logger
+import play.api.{Logger, Logging}
```
Comment (darrell-plant) [inline app/connectors/PropertyRepresentationConnector.scala:23]: usually `Logger` goes when you extend `Logging`

File: app/services/AgentRelationshipService.scala [hunk 2]
```diff
@@ -20,10 +20,11 @@ import binders.propertylinks.GetPropertyLinksParameters
 import connectors.PropertyRepresentationConnector
 import connectors.propertyLinking.PropertyLinkConnector
 import controllers.PaginationParams
+
 import javax.inject.{Inject, Named}
 import models.propertyrepresentation._
 import models.searchApi.{AgentPropertiesParameters, OwnerAuthResult}
-import play.api.Logger
+import play.api.{Logger, Logging}
```
Comment (darrell-plant) [inline app/services/AgentRelationshipService.scala:27]: imports

Comment (reddy-yattapu) [reply app/services/AgentRelationshipService.scala:27]: I am sorting out Logger imports now

File: project/plugins.sbt [hunk 3]
```diff
@@ -1,13 +1,13 @@
 resolvers += "HMRC-open-artefacts-maven" at "https://open.artefacts.tax.service.gov.uk/maven2"
 resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.15.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.3.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.1.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "2.0.0")
```
Comment (darrell-plant) [inline project/plugins.sbt:6]: I think `sbt-artifactory` can be removed: https://confluence.tools.tax.service.gov.uk/display/TEC/2021/05/05/New+version+of+sbt-auto-build

**Comment by darrell-plant** [comment]
LGTDz 👍 (with minor comments)

File: project/plugins.sbt (hunk 3 above)
Comment (darrell-plant) [reply project/plugins.sbt]: https://confluence.tools.tax.service.gov.uk/display/TEC/2021/01/18/Removing+the+sbt-artifactory+plugin

**Comment by habeeb01** [comment]
LGTM 👍

**Pull request merged**